Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9-alpine
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR app/
```

Comment:
The `WORKDIR` command should not include a trailing slash. It should be `WORKDIR app` instead of `WORKDIR app/`.